packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -99,15 +99,16 @@ export default class VTable {
      * Apply the given table format to this virtual table
      * @param format Table format to apply
      */
-    applyFormat(format: Partial<TableFormat>) {
+    applyFormat(
+        format: Partial<TableFormat>,

```

Comment:
Can we put everything into TableFormat, so no need to change the parameter here?

I suppose a tableFormat object should be self-contained and has everything we need.